<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontEdit</class>
 <widget class="QDialog" name="FontEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Шрифт:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Размер:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Цвет:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="colorComboBox"/>
   </item>
   <item>
    <widget class="QCheckBox" name="boldCheckBox">
     <property name="text">
      <string>Жирный</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="italicCheckBox">
     <property name="text">
      <string>Курсив</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="underlinedCheckBox">
     <property name="text">
      <string>Подчёркнутый</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
